cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+
+
+.. CDAP UI shows a session time out
+.. --------------------------------
+.. TBC.
+
+
+I don't see the CDAP Master service on YARN.
+--------------------------------------------
+- Ensure that the node where CDAP is running has a properly configured YARN client.
+- Ensure :ref:`YARN has enough memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.
+
+
+My CDAP Master log shows permissions issues.
+--------------------------------------------
+Ensure that ``hdfs:///#{hdfs.namespace}`` and ``hdfs:///user/#{hdfs.user}`` exist and are owned by ``#{hdfs.user}``.
+
+In rare cases, until `CDAP-3817 <https://issues.cask.co/browse/CDAP-3817>`__ is resolved,
+ensure ``hdfs:///#{hdfs.namespace}/tx.snapshot`` exists and is owned by ``#{hdfs.user}``. 
+
+In any other case, the error should show which directory it is attempting to access. Don't
+hesitate to ask for help at the `cdap-user@googlegroups.com <https://groups.google.com/d/forum/cdap-user>`__.
+
+
+My CDAP Master log shows an error about the dataset service not being found.
+----------------------------------------------------------------------------
+If you see an error such as::
+
+    2015-05-15 12:15:53,028 - ERROR [heartbeats-scheduler:c.c.c.d.s.s.MDSStreamMetaStore$1@71] 
+    - Failed to access app.meta table co.cask.cdap.data2.dataset2.DatasetManagementException: 
+    Cannot retrieve dataset instance app.meta info, details: Response code: 407, 
+    message:'Proxy Authentication Required', 
+    body: '<HTML><HEAD> <TITLE>Access Denied</TITLE> </HEAD>
+    
+According to that log, this error can be caused by a proxy setting. CDAP services
+internally makes HTTP requests to each other; one example is the dataset service.
+Depending on your proxy and its settings, these requests can end up being sent to the
+proxy instead.
+
+One item to check is that your system's network setting is configured to exclude both
+``localhost`` and ``127.0.0.1`` from the proxy routing. If they aren't, the services will
+not be able to communicate with each other, and you'll see error messages such as these.
+
+Where is the CDAP CLI (Command Line Interface)?
+-----------------------------------------------
+If you've installed the ``cdap-cli`` RPM or Deb, it's located under ``/opt/cdap/cli/bin``.
+You can add this location to your PATH to prevent the need for specifying the entire script every time.
+
+**Note:** These commands will list the contents of the package ``cdap-cli``, once it has
+been installed::
+
+  rpm -ql cdap-cli
+  dpkg -L cdap-cli
+
+.. _faq-installation-startup-memory-core-requirements:
+
+What are the memory and core requirements for CDAP?
+---------------------------------------------------
+The settings are governed by two sources: CDAP and YARN. The default setting for CDAP are
+found in the ``cdap-defaults.xml``, and are over-ridden in particular instances by the
```

Comment:
...and default YARN settings are found in... ??

This is a bit confusing and should probably be given it's own section under the installation docs, IMO. "Cluster Resources" or such...